src/webclients/components/sdfv/vscodeSdfv.ts

```diff
@@ -0,0 +1,1979 @@
+// Copyright 2020-2021 ETH Zurich and the DaCe-VSCode authors.
```

Comment:
could this file maybe split a little bit?

Reply:
addressed